bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -8,13 +9,9 @@ import Audio from '/imports/ui/components/audio-modal/component';
 let isUserPresenter = () => {
 
   // check if user is a presenter
-  let isPresenter = Users.findOne({
+  return isPresenter = Users.findOne({
```

Comment:
You can just return the `Users.findOne....` here. No point in having the `isPresenter = ...`